internal/interfaces/protocols_cifs_share.go

```diff
@@ -68,6 +69,7 @@ type ProtocolsCIFSShareResourceBodyDataModelONTAP struct {
 	ShowSnapshot          bool   `mapstructure:"show_snapshot,omitempty"`
 	UnixSymlink           string `mapstructure:"unix_symlink,omitempty"`  // API complains if this is not omit empty
 	VscanProfile          string `mapstructure:"vscan_profile,omitempty"` // API complains if this is not omit empty
+	AccessBasedEnumeration *bool `mapstructure:"access_based_enumeration,omitempty"`
```

Comment:
don't set boolean to omitempty, imagine the current state is true, and you want to set it to false. It will be omitted because it's false, and thus never showed up in the API body.